fileutil/fileutil_test.go

```
@@ -60,66 +59,58 @@ func TestWorkDir(t *testing.T) {
-		t.Errorf("Expected %v, got %v", expected, result)
-	}
+	result, err = ListFiles(tempDir, "[invalid]", "")
+	assert.NoError(err)
+	assert.Len(result, 0)
 
-	result, err = ListFiles(tempDir, "[invalid]")
-
-	if err != nil {
-		t.Fatalf("Unexpected error: %v", err)
+	// Test with skipExt option
+	expected = []string{
```

golangci-lint (1.64.8)

ineffectual assignment to expected

(ineffassign)